pkgs/applications/graphics/tesseract/get-language-hashes.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Usage:
+#   ./get-language-hashes.sh [--local-langs] [<tessdataRev>]
+#
+#   --local-langs
+#        Only use languages already defined in ./languages.nix instead of
+#        fetching all available languages from the tessdata repo
+#
+# Output:
+#   eng = "05gv...";
+#   fra = "0c60...";
+#   ...
+
+set -e
+
+tessdataRev=3cf1e2df1fe1d1da29295c9ef0983796c7958b7d
+
+for arg in "$@"; do
+    if [[ $arg == --local-langs ]]; then
+        localLangs=1
+    else
+        tessdataRev=$arg
+    fi
+done
+
+if [[ $localLangs ]]; then
+  langCodes=$(nix eval --raw '(
+    with (import ../../../.. { config = {}; overlays = []; });
+    let
+      languages = (pkgs.callPackage ./languages.nix {}).v3;
```

Comment:
Should we allow overriding version for the language list?

Reply:
Thought about that, too, but decided to not make this any more complex.

`--local-langs` will never be used by maintainers. We always want to fetch all languages when updating the tessdata version, 

`--local-langs` is more of a hint for users who want to build a custom language set, not a fully fleshed-out feature.

Reply:
Then maybe allow passing a space-separated list of languages as a single argument? All that current-Nixpkgs-list extraction magic seems to lack a use-case…

Reply:
You're right, let's keep this simple.